script/eslint.ts

```diff
@@ -26,7 +26,8 @@ const client: IClient = require('eslint_d/lib/client')
 const ESLINT_ARGS = [
   '--cache',
   '--rulesdir=./eslint-rules',
-  './{script,eslint-rules}/**/*.{j,t}s?(x)',
+  './eslint-rules/**/*.{j,t}s?(x)',
+  './script/**/*.ts',
```

Comment:
These have been split up to ensure the generated JS files under `script/` aren't linted.

Reply:
Nevermind, this isn't necessary any more.